src/coreclr/inc/gcmsg.inl

```diff
@@ -41,12 +41,33 @@
         return "%d gc thread waiting... Done";
     }
 
-    static const char* gcDetailedStartMsg()
+#define GC_DETAILED_START_PREFIX "*GC* %d(gen0:%d)(%d)"
+#define GC_DETAILED_START_STRESSLOG "(alloced for %.3fms, g0 %zd (b: %zd, %zd/h) (%.3fmb/ms), g3 %zd (%.3fmb/ms), g4 %zd (%.3fmb/ms))(%s)(%d)(%d)"
+#define GC_DETAILED_START_DPRINTF_EXTRA "(heap size: %.3fmb max: %.3fmb)"
+
+    static const char* gcDetailedStartPrefix()
+    {
+        STATIC_CONTRACT_LEAF;
+        return GC_DETAILED_START_PREFIX;
+    }
+
+    static const char* gcDetailedStartMsg(bool compatibleWithStressLog)
     {
         STATIC_CONTRACT_LEAF;
-        return "*GC* %d(gen0:%d)(%d)(alloced for %.3fms, g0 %zd (b: %zd, %zd/h) (%.3fmb/ms), g3 %zd (%.3fmb/ms), g4 %zd (%.3fmb/ms))(%s)(%d)(%d)(heap size: %.3fmb max: %.3fmb)";
+        if (compatibleWithStressLog)
+        {
+            return GC_DETAILED_START_PREFIX GC_DETAILED_START_STRESSLOG;
```

Comment:
Can you update the GCSTART message at the following spot in StressLogAnalyzer to match the message format returned here?

https://github.com/dotnet/runtime/blob/main/src/tools/StressLogAnalyzer/src/InterestingStringFinder.cs#L28

Once that's updated, then you'll have the expected experience.